x-pack/plugins/cases/public/components/case_view/case_view_tabs.tsx

```diff
@@ -46,7 +48,42 @@ const FilesTab = ({
   </>
 );
 
+const AlertsTab = ({
+  activeTab,
+  totalAlerts,
+  isLoading,
+  isExperimental,
+}: {
+  activeTab: string;
+  totalAlerts: number | undefined;
+  isLoading: boolean;
+  isExperimental: boolean;
+}) => (
+  <>
+    {ALERTS_TAB}
+    {!isLoading && (
+      <StyledNotificationBadge
+        data-test-subj="case-view-files-stats-badge"
+        color={activeTab === CASE_VIEW_PAGE_TABS.ALERTS ? 'accent' : 'subdued'}
+      >
+        {totalAlerts || 0}
+      </StyledNotificationBadge>
+    )}
+    {!isLoading && isExperimental && (
```

Comment:
The total alerts logic is independent of `isLoading` I think we can display one without the other

```suggestion
    {isExperimental && (
```